### ISSUE TITLE
Performance review on RDF4J/Blazegraph triplestore comparison

### ISSUE THREAD

**Issue by ericstephan** [GRIDAPPSD/GOSS-GridAPPS-D#533]

**Comment by ericstephan** [comment]
@tdtalbot @ronmelton I spent about a half hour looking around and after looking at Brad Beeb's (Blazegraph CEO) linkedin page and the news report.  This confirms what we hear in late June.  

https://www.i-programmer.info/news/84-database/11358-amazon-neptune-graph-database.html

**Comment by ericstephan** [comment]
@tdtalbot @telsethag below is a proposed evaluation list for our meeting today based on the original alternatives analysis.  I added a few other items such as query response time, bulkload, MRMW support. 

I've found https://db-engines.com/en/ to be helpful answering some of the information.  In Google search if you type in "Jena vs Blazegraph" the db-engines.com provides a side by side comparison between products.


Proposed Triplestore Evaluation
Date: July 17, 2018

Goal:  Evaluate candidate Triplestores as an alternate to Blazegraph.

Candidates:
•	Fuseki/Jena
•	RDF4J
•	Virtuoso
•	Others?


1.	Administration
1.0	Lightweight admin requirements
1.1	Docker or containerization support for easy deployment
1.2	DBMS Authentication
1.3	DBMS Authorization
1.4	User account setup tools
2.	Management
2.0	Administrative free schema maintenance
2.1	Ability to intuitively represent CIM objects
2.2	Schema evolution free solution
2.3	Value type and referential integrity checking 
2.4	Indexing
2.5	Supports data management language
2.6	Data partitioning
2.7	Directed graph
2.8	Polymorphism
3.	Product Stability, Standards Adherence and Licensing
3.0	Ease of technology transfer (within tech family)
3.1	Community involvement
3.2	Product maturity
3.3	BSD equivalent open source license support
4.	Bulk loading and Writing
4.0	Transactions
4.1	Rollback
4.2	Ease of CIM RDF/XML ingests from use cases
4.3	Multi-write
4.4	Bulk load speed test
5.	Access
5.0	Standardized query language
5.1	Multi-read
5.2	Path traversal
5.3	Relational algebraic 
5.4	Polymorphism
5.5	Data structure translation
5.6	Special functions (aggregation, count, sum)
5.7	REST access
5.8	client libraries
5.9	Query speed
